x-pack/plugins/ml/server/routes/system.ts

```diff
@@ -247,4 +247,41 @@ export function systemRoutes(
       }
     })
   );
+
+  /**
+   * @apiGroup SystemRoutes
+   *
+   * @api {get} /api/ml/index_exists/<indexName> check if index exists
+   * @apiName MlSpecificIndexExists
+   * @apiDescription Returns boolean for if index exists
+   */
+  router.get(
```

Comment:
is this new endpoint needed? Couldn't the `index_exists` endpoint we already have be used?
Or could the original be change to use the `exists` api, rather than field caps.
It looks like the original probably should just use `exists` rather than field caps.

Reply:
If you switch to using the current `index_exists` endpoint in `system.ts`, could you edit the comment on that `api` to `Check if index exists` from the current `ES Field caps wrapper checks if index exists` text, which doesn't make it obvious in the rendered docs what that endpoint does (and is therefore easy to miss).

![image](https://user-images.githubusercontent.com/7405507/144631874-0067c500-35c1-4950-bf5c-77f84d4a396d.png)

Reply:
@jgowdyelastic - good call - I'll go ahead and replace the existing index_exists  (wrapping the fields api) with the one I created since I agree it should be using the exists endpoint 👍

Reply:
Updated index_exists to use the exists endpoint - c222f690f647352fac72626d8a215bdd0acf63d8